### PR TITLE
[TD-23] Adding logic to filter out polls that begin with the words 'Single message' regardless of case.

### PR DESCRIPTION
**Pull request by emullaney**

### CODE REVIEW

File: tracpro/polls/models.py [hunk 1]
```diff
@@ -71,8 +71,13 @@ def set_active_for_org(self, org, uuids):
...
-        temba_polls = {p.uuid: p for p in temba_polls}
+        temba_polls_result = get_client(org).get_flows()
+
+        # Filter out polls with names starting with 'Single Message'
+        temba_polls = {}
+        for poll in temba_polls_result:
+            if poll.name.lower()[:14] != 'single message':
+                temba_polls[poll.uuid] = poll
 
         # Remove Polls that are no longer on RapidPro.
```
Comment (dpoirier) [inline tracpro/polls/models.py]: Maybe add to this comment, ", or that we're filtering out".?

File: tracpro/polls/models.py [hunk 2]
```diff
@@ -71,8 +71,13 @@ def set_active_for_org(self, org, uuids):
...
         """Update the org's Polls from RapidPro."""
         # Retrieve current Polls known to RapidPro.
-        temba_polls = get_client(org).get_flows()
-        temba_polls = {p.uuid: p for p in temba_polls}
+        temba_polls_result = get_client(org).get_flows()
+
+        # Filter out polls with names starting with 'Single Message'
+        temba_polls = {}
+        for poll in temba_polls_result:
+            if poll.name.lower()[:14] != 'single message':
```
Comment (dpoirier) [inline tracpro/polls/models.py]: I only see "Single Message" in the examples, is there any need to make this case-insensitive?  Maybe we could just say ```if not poll.name.startswith("Single Message")```?

File: tracpro/polls/models.py (hunk 1 above)
Comment (emullaney) [inline tracpro/polls/models.py]: Sure.

File: tracpro/polls/models.py (hunk 2 above)
Comment (emullaney) [inline tracpro/polls/models.py]: poll isn't a Django queryset object but rather a tembaclient Flow object, so startswith isn't available. 

I could remove the lower() if we don't think it's necessary, that's one less string operation...

Comment (dpoirier) [inline tracpro/polls/models.py]: startswith is a Python string operation, so should be available on any string.

Comment (emullaney) [inline tracpro/polls/models.py]: Oh cool! I didn't know that one. Just added it.

**Comment by dpoirier** [comment]
+1

**Pull request merged**